prow/cmd/deck/static/user-script.js

```diff
@@ -669,3 +663,23 @@ function createIcon(iconString, tooltip = "", styles = [], isButton = false) {
 
     return container;
 }
+
+/**
+ * Create a simple message with an icon.
```

Comment:
we aren't setting an icon anymore?

Reply:
The login message needs an :smile: icon (or not).